src/components/MediaPlayer/index.ts

```diff
@@ -65,6 +65,16 @@ export class MediaPlayer implements RedomComponent {
     return this.nativePlayerElement.paused;
   }
 
+  public get volume(): number {
+    return this.nativePlayerElement.volume;
+  }
+
+  public set volume(volume: number) {
+    this.nativePlayerElement.volume = volume;
+    const volumeSlider = this.el.querySelector('input.volume-slider') as HTMLInputElement;
```

Comment:
The UI update should be done using volumechange event on the native player element.